src/code/utilities/lang/en-us.js

```diff
@@ -13,7 +13,7 @@ export default {
 
   // Challenge popup alerts
   "~ALERT.TITLE.GOOD_WORK": "Good work!",
-  "~ALERT.TITLE.MISSION_ACCOMPLISHED": "Mission Accomplished!",
+  "~ALERT.TITLE.MISSION_ACCOMPLISHED": "Level Accomplished!",
```

Comment:
As we discussed elsewhere, if the plan is not to have the flat level structure beyond the classroom tests, then there should be a story somewhere for reintroducing the level/mission structure which includes changing these strings back along with updating the authoring document.